### ISSUE TITLE
Named bookmarks

### ISSUE THREAD

**Issue by hmsww** [JeffryBooher/brackets-bookmarks-extension#19]
The ability to name the bookmarks would be epic!! Please!

**Comment by intox9** [comment]
Agreed :-]

**Comment by SheepDomination** [comment]
Agree again :) Please!

**Comment by mitralone** [comment]
Or at least make the bookmarked line display in the bookmarks pane instead of the address. This way we can just bookmark a comment line and have the ability automatically.

**Comment by hmsww** [comment]
Agree.

On Wednesday, October 14, 2015, Tufan ALIN <notifications@github.com> wrote:

> Or at least make the bookmarked line display in the bookmarks pane instead
> of the address. This way we can just bookmark a comment line and have the
> ability automatically.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JeffryBooher/brackets-bookmarks-extension/issues/19#issuecomment-147954014>
> .
>


-- 
Sent from Gmail Mobile

**Comment by SheepDomination** [comment]
Agreed with the extra "d" which has got to add major weight for the developer to want to add this feature. ;-)

**Comment by JeffryBooher** [comment]
I'll have a little time the week ending with halloween to do it... I'll try and get it done...

**Comment by SheepDomination** [comment]
Thanks Jeffry, I and the rest look forward to it.

**Comment by artquest** [comment]
Line number... so informative. 
Just useless.
